Gemfile

```diff
@@ -6,12 +6,13 @@ gemspec
 gem 'engine_cart'
 
 gem 'rubocop', require: false
-
+gem 'yajl-ruby', require: 'yajl'
```

Comment:
Yes. The gem is included in the gemspec as well, but it gets required using `require 'yajl'`, not `require 'yajl-ruby'`.